src/shell/bash_functions

```diff
@@ -125,9 +125,11 @@ hrfs() {
 
             }
         }' |
-    sed -e ":l; s/\([0-9]\)\([0-9]\{3\}\)/\1,\2/; t l"
-    #                          └─ add commas to the numbers
-    #                             (changes "1023.2 KB" to "1,023.2 KB")
+    sed -e ":l"
+        -e "s/\([0-9]\)\([0-9]\{3\}\)/\1,\2/"
```

Comment:
@wingy3181 Missing `\` at the end. Should be: `-e "s/\([0-9]\)\([0-9]\{3\}\)/\1,\2/" \`